src/model/CoilHeatingWaterToAirHeatPumpEquationFit.cpp

```diff
@@ -383,6 +386,32 @@ namespace model {
       }
     }
 
+    ComponentType CoilHeatingWaterToAirHeatPumpEquationFit_Impl::componentType() const {
+      return ComponentType::Heating;
+    }
+
+    std::vector<FuelType> CoilHeatingWaterToAirHeatPumpEquationFit_Impl::coolingFuelTypes() const {
+      return {};
+    }
+
+    std::vector<FuelType> CoilHeatingWaterToAirHeatPumpEquationFit_Impl::heatingFuelTypes() const {
+      std::set<FuelType> result;
+      result.insert(FuelType::Electricity);
+      if (auto p_ = plantLoop()) {
+        return p_->heatingFuelTypes();
+      }
+      return {};
+    }
+
+    std::vector<AppGFuelType> CoilHeatingWaterToAirHeatPumpEquationFit_Impl::appGHeatingFuelTypes() const {
+      std::set<AppGFuelType> result;
+      result.insert(AppGFuelType::HeatPump);
+      if (auto p_ = plantLoop()) {
+        return p_->appGHeatingFuelTypes();
+      }
+      return {};
+    }
```

Comment:
I'm not 100% sure I did the WaterToAir / AirToWater heat pump coils correctly to be honest, a review would be nice.